deckhouse-controller/pkg/apis/deckhouse.io/v1alpha1/module_source.go

```diff
@@ -104,10 +106,11 @@ type ModuleSourceStatus struct {
 }
 
 type AvailableModule struct {
-	Name       string `json:"name"`
-	Version    string `json:"version,omitempty"`
-	Policy     string `json:"policy,omitempty"`
-	Checksum   string `json:"checksum,omitempty"`
-	PullError  string `json:"pullError,omitempty"`
-	Overridden bool   `json:"overridden,omitempty"`
+	Name         string `json:"name"`
+	Version      string `json:"version,omitempty"`
+	Policy       string `json:"policy,omitempty"`
+	Checksum     string `json:"checksum,omitempty"`
+	ProcessError string `json:"processError,omitempty"`
```

Comment:
PullError naming was a mistake. we must unify this field, but not delete PullError now (because of contract)
```suggestion
	Error string `json:"error,omitempty"`
```